README.md

```diff
@@ -1,15 +1,59 @@
-# Team members present: 
-## What's this PR do? 
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] Styling
-- [ ] Testing
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-## Where should the reviewer start? 
-## How should this be manually tested? 
-## Any background context you want to provide? 
-## What are the relevant tickets? 
-* Addresses:
-* Closes:
-## Screenshots (if appropriate) 
-## Questions:
\ No newline at end of file
+## House Rules
+
+**Contributors**
+
+[Hanna Kim](https://github.com/hannakim91), [Michael Walker](https://github.com/MichaelEWalker87), [Katy St Sauveur](https://github.com/krogowsk531)
+
+**Project Description**
+
+*A site for disputing game rules where a user can select a game, get information about that game and fill out a form that allows them to dispute a game rule. The rule dispute will persist through local storage in each individual game page so that a user can debate rules for the specified game.
+
+**Installation**
+
+**Fork this repository:**
+
+* https://github.com/krogowsk531/house-rules
+
+**Clone your forked repository**
+
+* `git clone` and the copied URL
+
+**Change into the directory and install the project dependencies**
+
+* `cd` into directory and run `npm install` for dependencies
+
+**Project Goals**
+
+* Create a functioning app while learning a new technology
+* Decide what the most viable product for the app will be
+* Use an external API of the teams choosing
+* The data from the API must be able to be manipulated by the user
+* To have a multipage app using routing
+
+**Wins**
+
+* Learning and working with Next.js in a collaborative environment with a set deadline
+* Incorporating Next.js features such as dynamic routing and server side rendering
+* Able to integrate Hooks and a complicated testing suite
```

Comment:
^incl custom hooks; maybe unfamiliar vs complicated testing suite? either is good though!

not sure if it would be good to explain how hooks help us keep components functional (more legible/straightforward code - not sure if it runs faster too?)

Reply:
Done